### DISCUSSION TITLE
Why does snowflake_grant_privileges_to_database_role have a green checkmark ?

### DISCUSSION THREAD

**Discussion by brian-franklin-kr** [Snowflake-Labs/terraform-provider-snowflake#2239] [General] [answered]
Why does snowflake_grant_privileges_to_database_role have a green checkmark ?

snowflake_grant_privileges_to_database_role doesn't yet appear in the code as even being started

**Comment by sfc-gh-asawicki** [comment, marked as answer]
Hi @brian-franklin-kr. Thanks for the question.

There are two columns:
- `SDK migration status`, which is marked with a green checkmark; this is the status of supporting the new logic for the given snowflake object NOT using it in resource/datasource
- `Integration status`, which indicates if the given resource/datasource is using a new SDK; this one is marked as in progress.
